net-nds/phpldapadmin/phpldapadmin-1.2.6.6.ebuild

```diff
@@ -7,29 +7,39 @@ inherit webapp
 
 MY_PN="phpLDAPadmin"
 DESCRIPTION="phpLDAPadmin is a web-based tool for managing all aspects of your LDAP server"
-HOMEPAGE="http://phpldapadmin.sourceforge.net"
+HOMEPAGE="https://github.com/leenooks/phpLDAPadmin"
 SRC_URI="https://github.com/leenooks/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="GPL-2"
 KEYWORDS="~alpha ~amd64 ~ia64 ~ppc ~sparc ~x86"
-IUSE=""
 
 RDEPEND="
-	>=dev-lang/php-7.2[hash(+),ldap,session,xml,nls]
+	>=dev-lang/php-8.0[hash(+),ldap,session,xml,nls]
+	dev-libs/openssl:=
 	virtual/httpd-php
 "
+BDEPEND="
+	media-libs/libpng
+"
 S="${WORKDIR}/${MY_PN}-${PV}"
 
-# http://phpldapadmin.git.sourceforge.net/git/gitweb.cgi?p=phpldapadmin/phpldapadmin;a=commit;h=7dc8d57d6952fe681cb9e8818df7f103220457bd
 PATCHES=(
 	"${FILESDIR}/${PN}-1.2.1.1-fix-magic-quotes.patch"
+	"${FILESDIR}/${PN}-1.2.6.2-r1-default-templates.patch"
+	"${FILESDIR}/${PN}-1.2.6.4-getDN-htmlspecialchars.patch"
```

Comment:
This looks like a small enough bugfix, I will merge it but please sent it upstream too.

Reply:
Note: The file `lib/HTMLTree.php` that si changed by this patch was removed in this upstream commit: https://github.com/leenooks/phpLDAPadmin/commit/5b046a95eb3cfd696b0f9434b99792dab37022a0

So future versions won't need this patch anymore.